README.md

```diff
@@ -29,65 +24,33 @@ Before you start developing your own integration, you might benefit from looking
 * the [Acrolinx SDKs](https://github.com/acrolinx?q=sdk), and
 * the [Acrolinx Demo Projects](https://github.com/acrolinx?q=demo).
 
-### Start Developing
+## Getting Started
 
-#### Installation
+### Build the Project
 
-##### Maven
+1. You need Java 11 to build this project.
+2. This project uses [Gradle](https://gradle.org/).
+To build this project with the [Gradle Wrapper](https://docs.gradle.org/current/userguide/gradle_wrapper.html#sec:using_wrapper), execute the following command:
 
-```xml
-<dependency>
-    <groupId>com.acrolinx.client</groupId>
-    <artifactId>sdk</artifactId>
-    <version>1.0.21</version>
-</dependency>
+```bash
+./gradlew build
 ```
 
-##### Gradle
-
-```groovy
-repositories {
-    mavenCentral()
-}
+on an UNIX system, or
```

Comment:
The right article to be used  before the word UNIX is "a" because the U is pronounced as "yoo".

Reply:
Fixed!